packages/appstore/src/modules/onboarding/__test__/onboarding.spec.tsx

```diff
@@ -0,0 +1,231 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import Onboarding from '../onboarding';
+import { getTradingHubContents } from 'Constants/trading-hub-content';
+import EmptyOnboarding from '../empty-onboarding';
+
+describe('Onboarding', () => {
+    it('should render <Onboarding /> component', () => {
+        const mock = mockStore({
```

Comment:
@aizad-deriv You don't need to override the default values if you don't want to change them, Please double check `mockStore` in each test case and remove the default ones 🙏🏻